File: lib/go/rpc/RequestMsg.go
package rpc

const RequestMsg_strFingerprint = "e61c37380db853addbaafae6c4850b2e"

type RequestMsg struct {
	Header   RpcHeader `json:"header"`
	MsgSeqId int64     `json:"msgSeqId"`
	MsgId    int32     `json:"msgId"`
	Buff     []byte    `json:"buff"`
}

func (this *RequestMsg) GetFingerprint() string {
	return RequestMsg_strFingerprint
}
func (this *RequestMsg) Serialize(P__ IProtocol) {
	this.Header.Serialize(P__)

	P__.WriteInt64(this.MsgSeqId)

	P__.WriteInt32(this.MsgId)

	P__.WriteBinary(this.Buff)

}
func (this *RequestMsg) DeSerialize(P__ IProtocol) error {
	this.Header.DeSerialize(P__)

	this.MsgSeqId = P__.ReadInt64()

	this.MsgId = P__.ReadInt32()

	this.Buff = P__.ReadBinary()

	return nil
}

File: lib/ts/rpc/RequestMsg.ts
import  * as rpc from "../rpc/index"
import  {RpcHeader}  from "./RpcHeader"
export class RequestMsg
{ 
	public static readonly strFingerprint:string="e61c37380db853addbaafae6c4850b2e"
	public header:RpcHeader=new RpcHeader()
	public msgSeqId:rpc.Int64=new rpc.Int64
	public msgId:number=0
	public buff!:Uint8Array
	//serialize
	public serialize( __P__:rpc.Protocol):void 
	{ 
		this.header.serialize(__P__);

		__P__.writeInt64(this.msgSeqId);

		__P__.writeInt32(this.msgId);

		__P__.writeBinary(this.buff);

	}// serialize
	//deSerialize
	public deSerialize( __P__:rpc.Protocol):void 
	{ 
		this.header.deSerialize(__P__)

		this.msgSeqId=__P__.readInt64()

		this.msgId=__P__.readInt32()

		this.buff=__P__.readBinary()

	}// deSerialize
}//class
